src/System.IO.Compression.ZipFile/src/packages.config

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Yep, thank you, this branch doesn't have Eric's changes merged in yet.  Let me do that now and update.